architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -147,6 +147,18 @@ maintain a uniform odd number of hosts when collocating master services and
 etcd.
 endif::[]
 
+[NOTE]
+====
+In production {product-title} clusters, you must maintain high availability
+of the API Server load balancer. If the API Server load balancer is not
+available, nodes cannot report their status, all their pods are marked dead,
+and the pods' endpoints are removed from the service.
+
+You must separately configure HA for the API Server load balancer. To configure
+HA for it, you can run multiple HAProxy load balancers and use Keepalived to
```

Comment:
Maybe s/To configure HA for it,/To do this,

We use this in other parts of the docs.

But also, is there somewhere in the docs that explains how to to this? Maybe link out to there?

Reply:
Thanks, @bfallonf! The source bug says that it's not something that you do in the product, that it's completely separate. There's nothing in the docs for it, and while this one method of configuring 
How's this?
 "In addition to configuring HA for {product-title}, you must separately configure HA for the API Server load balancer. One HA option is to run multiple HAProxy load balancers and use Keepalived to provide a floating virtual IP address for them."

Reply:
@kalexand-rh Hmm sure. I think I have a BZ about creating docs for this maybe... +1 to your suggestion.

Reply:
Well, if you have a BZ about it, then I guess we do need to be opinionated. :)
I'll update this change and merge. Thanks @bfallonf!